src/ashley/models.py

```diff
@@ -151,6 +157,19 @@ def sync_user_groups(self, user, roles: List[str]) -> None:
                 logger.debug("Add user %s to group %s", user, group)
                 user.groups.add(group)
 
+    def get_user_roles(self, user) -> List[str]:
+        """
+        Return list of roles name without group pattern
+        cg:{context.id}:role: for this user in this LTI Context
+        """
+        role_group_prefix = self.get_group_role_name("")
+        user_role_groups = user.groups.filter(
+            name__startswith=role_group_prefix
+        ).values_list("name", flat=True)
+        roles = [group.replace(role_group_prefix, "") for group in user_role_groups]
+        logger.debug("List of roles %s", roles)
```

Comment:
If you want to keep this debug, I suggest you to include the user in the logged string, otherwise it's a difficult to do something useful with this information.